domains/authorization/Authorization.API/API.IntegrationTests/API.IntegrationTests/Setup/IntegrationTestFixture.cs

```diff
@@ -92,6 +95,37 @@ public new async Task DisposeAsync()
         await PostgresContainer.DisposeAsync();
     }
 
+    protected override IHost CreateHost(IHostBuilder builder)
+    {
+        var host = base.CreateHost(builder);
+        var authenticationSchemeProvider = host.Services.GetRequiredService<IAuthenticationSchemeProvider>();
```

Comment:
Wrap these next lines in a _ReplaceB2CAuthenticationSchemes_ method to make it clear what the purpose is.

Reply:
fixed